src/ServicePulse.Host/app/modules/configuration/views/platformconnection.html

```diff
@@ -0,0 +1,49 @@
+﻿<platform-trial-expired ng-if="isPlatformTrialExpired"></platform-trial-expired>
+<platform-expired ng-if="isPlatformExpired"></platform-expired>
+<upgrade-protection-expired ng-if="isInvalidDueToUpgradeProtectionExpired"></upgrade-protection-expired>
+
+<div class="container" ng-if="!isPlatformTrialExpired && !isPlatformExpired && !isInvalidDueToUpgradeProtectionExpired">
+    <div class="row">
+        <div class="col-sm-12">
+            <h1>Configuration</h1>
+        </div>
+    </div>
+</div>
+
+<div class="container" ng-if="!isPlatformTrialExpired && !isPlatformExpired && !isInvalidDueToUpgradeProtectionExpired">
+    <section name="platformconnection">
+
+        <configuration-tabs></configuration-tabs>
+
+        <div class="col-sm-12">
+            <div class="box">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <h3>Connect your endpoint to the Service Platform</h3>
```

Comment:
I think this should be Particular Service Platform. See the [brand style guide](https://github.com/Particular/DeveloperEducation/blob/master/Brand%20style%20guide/README.md#platform)